sentry_sdk/integrations/redis/__init__.py

```diff
@@ -218,6 +218,11 @@ def sentry_patched_execute_command(self, name, *args, **kwargs):
         if data_should_be_truncated:
             description = description[: integration.max_data_size - len("...")] + "..."
 
+        # TODO: Here we could also create the caching spans.
+        # TODO: also need to check if the `name` (if this is the cache key value) matches the prefix we want to configure in __init__ of the integration
+        #       Questions:
+        #       -) We should probablby have the OP.DB_REDIS span and a separate OP.CACHE_GET_ITEM (or set_item) span, right?
+        #       -) We probably need to research what redis commands are used by caching libs.
```

Comment:
Will these be addressed in this PR? If not, I would suggest linking an issue to each – the questions also should likely only go in the issue, not the comment.

Reply:
done